src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -115,6 +115,7 @@ Author(s):
     X(winrt::Windows::UI::Text::FontWeight, FontWeight, "weight", DEFAULT_FONT_WEIGHT) \
     X(IFontAxesMap, FontAxes, "axes")                                                  \
     X(IFontFeatureMap, FontFeatures, "features")                                       \
+    X(bool, CustomGlyphs, "customGlyphs", true)                                        \

```

Comment:
We may want to call this `compatibility.integratedDrawingGlyphs` or something - IMO and of course open to discussion!

Reply:
Feature label: Built-in Glyphs
Help text: "Use built-in glyphs for BoxDrawing, BlockElement and Powerline characters"

I think this fits well with the future plan where we would draw them even when the font itself may not come with any of those glyphs (Eg. unpatched fonts)

Reply:
I'm mostly worried about the JSON name - where there is no help text and "glyphs" is awfully broad (there are a looooottt of glyphs in a font!)

Reply:
I'm opposed to renaming this setting for 2 reasons:
* wezterm calls it `custom_block_glyphs`
* VS Code calls it `terminal.integrated.customGlyphs`

So while I agree that we should call it "integrated glyphs" I think consistency with other applications is more important to make it easier for users to find related settings in related applications.

Reply:
FWIW that's for "vscode's integrated terminal", not for "integrated custom glyphs"

That's fair, however.

Reply:
In case you're still considering other names, I think both Alacritty and Contour use something like `builtin_box_drawing` for this option (as a sub setting of the `font` configuration).